src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -66,6 +66,7 @@ namespace System.Collections
     // 
     [DebuggerTypeProxy(typeof(System.Collections.Hashtable.HashtableDebugView))]
     [DebuggerDisplay("Count = {Count}")]
+    [Serializable]
```

Comment:
More generally, are we actually exposing this Hashtable from any APIs, e.g. are we returning this from APIs that return IDictionary?  If so, we probably want to move the "real" Hashtable to CoreCLR anyway, to handle existing code that casts the IDictionary to Hashtable.